backend/src/index.ts

```diff
@@ -170,3 +173,15 @@ function addTime(time: number) {
 
     connectedClients.forEach(c => c.emit('addTime', time));
 }
+
+
+
+
+
+
+
+// -C user inputs time
+// -> addTime
+//   -S addTime()
+//     <- addTime
+//   -C updateTimer()
```

Comment:
```suggestion
```
monkaW

Reply:
Noticed it as you submitted the review, fixed in a6466d0